From aa40aec097cd8ce3d58d617986ee46d1568cada7 Mon Sep 17 00:00:00 2001
From: Yuta Saito <kateinoigakukun@gmail.com>
Date: Wed, 7 Feb 2024 16:35:42 +0000
Subject: [PATCH] [wasm][build] Exclude Tools from build when BUILD_TOOLS is
 OFF

This is a follow up to 5e7281b993ca04486ed8f2add39c8b11f132a550, which
just adds the `BUILD_TOOLS` option but doesn't actually use it.
---
 Sources/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Sources/CMakeLists.txt b/Sources/CMakeLists.txt
index 94b85146..584d6bf0 100644
--- a/Sources/CMakeLists.txt
+++ b/Sources/CMakeLists.txt
@@ -4,4 +4,6 @@ if(BUILD_NETWORKING)
   add_subdirectory(FoundationNetworking)
 endif()
 add_subdirectory(FoundationXML)
-add_subdirectory(Tools)
+if(BUILD_TOOLS)
+  add_subdirectory(Tools)
+endif()
-- 
2.43.0

